x-pack/plugins/apm/public/components/alerting/rule_types/anomaly_rule_type/index.tsx

```diff
@@ -54,12 +54,12 @@ interface Props {
   metadata?: AlertMetadata;
   setRuleParams: (key: string, value: any) => void;
   setRuleProperty: (key: string, value: any) => void;
+  errors: { anomalyDetectorTypes?: string };
```

Comment:
What about:

```suggestion
  hasAnomalyDetectorTypeError: boolean
```

Reply:
I need to use the errors object that comes with the props